charts/tfy-agent/templates/tfy-agent-proxy-clusterrole-cs.yaml

```diff
@@ -15,13 +15,25 @@ rules:
     resources: ["storageclasses"]
     verbs: ["get", "list"]
 
+  - apiGroups: [""]
```

Comment:
These should be present on 
https://github.com/truefoundry/infra-charts/blob/efbb1660fa54f8e725ee8ff768e8b220703c4857/charts/tfy-agent/templates/tfy-agent-proxy-clusterrole-ns.yaml

Reply:
Read https://github.com/truefoundry/infra-charts/tree/efbb1660fa54f8e725ee8ff768e8b220703c4857/charts/tfy-agent#namespace-scope-resource-access